### DISCUSSION TITLE
Saving m2m

### DISCUSSION THREAD

**Discussion by JaapGroot** [uptrace/bun#98] [Q&A]
Hi,

I was wondering how I should approach saving a m2m relation in Bun.

Thanks,

**Comment by vmihailenco** [comment]
Did you see https://github.com/uptrace/bun/tree/master/example/many-to-many?

**Reply by JaapGroot** [reply]
Did not notice the insert at the bottom, I did something like that in my program.

I previously used GORM, but that did not suit my needs. Was wondering if there was another method that I was not seeing.

Thanks for the reply!
